dev_guide/new_app.adoc

```diff
@@ -21,60 +21,62 @@ and converts it into an image that can run inside of a pod. The images are
 deployed via a deployment configuration, and a service is hooked up to the first
 public port of the application.
 
-NOTE: If you specify source code, you may need to
+[NOTE]
+====
+If you specify source code, you may need to
 link:builds.html#starting-a-build[run a build] with `osc start-build` after the
 application is created.
+====
 
 == Using the CLI
 
-To try to create an application based on the source code in the current
-directory, run:
+To create an application based on the source code in the current
+directory:
 
 ----
 $ osc new-app .
 ----
 
-To use the public DockerHub MySQL image to create an application, run:
+To use the public DockerHub MySQL image to create an application:
```

Comment:
"DockerHub MySQL image" here could also link directly to https://registry.hub.docker.com/_/mysql/ for some added context.